framework/src/main/java/com/github/dbmdz/flusswerk/framework/messagebroker/ConnectionConfig.java

```diff
@@ -1,15 +1,59 @@
 package com.github.dbmdz.flusswerk.framework.messagebroker;
 
 import com.rabbitmq.client.Address;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
-public interface ConnectionConfig {
+class ConnectionConfig {
 
-  String getUsername();
+  private String username;
 
-  String getPassword();
+  private String password;
 
-  String getVirtualHost();
+  private String virtualHost;
 
-  List<Address> getAddresses();
+  private final List<Address> addresses;
+
+  ConnectionConfig() {
+    addresses = new ArrayList<>();
+    setPassword("guest");
+    setUsername("guest");
+    setVirtualHost("/");
+  }
+
+  public String getUsername() {
+    return username;
+  }
+
+  public final void setUsername(String username) {
+    this.username = username;
+  }
+
+  public String getPassword() {
+    return password;
+  }
+
+  public final void setPassword(String password) {
+    this.password = password;
+  }
+
+  public String getVirtualHost() {
+    return virtualHost;
+  }
+
+  public final void setVirtualHost(String virtualHost) {
+    this.virtualHost = virtualHost;
+  }
+
+  public void addAddress(String host, int port) {
+    addresses.add(new Address(host, port));
+  }
+
+  public List<Address> getAddresses() {
```

Comment:
Braucht es kein `setAddresses`?

Reply:
Dafür wird `addAddress` verwendet. Ich habe für diesen MR tatsächlich nur Code entfernt (bis auf eine Methdoe, die ich aus dem gelöschten Interface in die Klasse verschoben hatte).

Reply:
Ich dachte nur, dass es für den Fall, dass mehrere Adressen gesetzt werden soll, ein `setAddresses` mit einer Liste von Adressen vielleicht sinnvoller wäre, als mehrmals `addAddress` aufzurufen. Aber geht natürlich auch so :+1:

Reply:
Das ist ein Teil des Frameworks, den wir wahrscheinlich nochmal ein bisschen (oder auch ein bisschen mehr) überarbeiten werden. Allerdings erst, wenn wir wissen wo die Reise an dieser Stelle hingeht.

Reply:
Danke für die Erläuterung!